flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -425,6 +426,8 @@ static ContainerLaunchContext createTaskExecutorContext(
 
         ctx.setEnvironment(containerEnv);
 
+        setAclsFor(ctx, flinkConfig);
```

Comment:
Is this to set `acls` to `TaskManager` containers and the `YarnClusterDescriptor` is for `AM` container?

Reply:
This is within `createTaskExecutorContext` - so for TM.
The change in `YarnClusterDescriptor` is within `startAppMaster`, so for JM